Sources/TuistGenerator/Utils/SwiftPackageManagerInteractor.swift

```diff
@@ -48,7 +71,8 @@ public class SwiftPackageManagerInteractor: SwiftPackageManagerInteracting {
             return
         }
 
-        let rootPackageResolvedPath = path.appending(component: ".package.resolved")
+        guard let rootDirectory = rootDirectoryLocator.locate(from: path) else { throw SwiftPackageManagerInteractorError.rootDirectoryNotFound(path) }
```

Comment:
It's worth falling back to the current `path` (previous behaviour) to avoid failing in the event there is no Tuist root directory which is currently optional (its valid to have a bare bones directory with `Project.swift`, no `Tuist` folder and not be within a git repo 🤐)

e.g. When I copy a fixture to `/tmp/` without a `Tuist` folder:

```
$ tuist generate
Generating workspace App.xcworkspace
Generating project App
Couldn't locate the root directory from path /private/tmp/Testing. The root directory is the closest directory that contains a Tuist or a .git directory.
```

Reply:
That makes sense, thanks for the feedback, I added `path` as a fall back 👍